test/rulesets/Random/random.jl

```diff
@@ -1,3 +1,9 @@
+struct Normal
```

Comment:
Maybe since this is not in a package called distbrutions we can be more explictit what it is?
```suggestion
# Simple Distributions like object for testing purposes
struct NormalDistribution
```